lib/logasm/adapters/stdout_adapter.coffee

```diff
@@ -1,23 +1,24 @@
 winston = require('winston')
-_  = require 'underscore'
+buildEvent = require('../event')
 
 class StdoutAdapter
-  constructor:(level) ->
+  constructor:(level, _serviceName, options) ->
     @logger = new winston.Logger
     @logger.add winston.transports.Console,
-      timestamp: ->
-        "#{new Date().toISOString()} ##{process.pid}"
+      json: Boolean(options.json)
+      timestamp: !Boolean(options.json) # Json adds @timestamp manually
       level: level
       colorize: true
+      stringify: (obj) -> JSON.stringify(obj)
```

Comment:
`json: true` has `JSON.stringify(obj, null, 2)` which means it creates multiline json objects. We need one-line logs.